mix.exs

```diff
@@ -11,7 +11,8 @@ defmodule Divo.MixProject do
       package: package(),
       docs: docs(),
       description: description(),
-      source_url: "https://github.com/smartcitiesdata/divo"
+      source_url: "https://github.com/smartcitiesdata/divo",
+      dialyzer: [plt_add_apps: [:mix]]
```

Comment:
```elixir
dialyzer: [plt_add_apps: [:mix], plt_file: {:no_warn, "path/to/dialyzer.plt"}]
```

This is how you configure the cached path. But the trick is, this path has to exist before `mix dialyzer` will run.